### PR TITLE
fix(18448): recolor selected area point geometry

### PR DESCRIPTION
**Pull request by alesiahil**
Tasks:
https://kontur.fibery.io/Tasks/Task/Recolor-point-geometry-into-blue-for-Selected-area-18448
https://kontur.fibery.io/Tasks/Task/Fix-labels-and-texts-on-Kontur-apps-18566

### CODE REVIEW

File: src/features/focused_geometry_layer/renderers/FocusedGeometryRenderer.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { UserGeometryRenderer } from '~core/logical_layers/renderers/UserGeometryRenderer';
-import Icon from '../icons/marker_black.png';
```
Comment (albaranau) [inline src/features/focused_geometry_layer/renderers/FocusedGeometryRenderer.ts:2]: The `marker_black.png` file itself doesn't seem to be used anywhere after this change. Would be better to delete it

Comment (alesiahil) [reply src/features/focused_geometry_layer/renderers/FocusedGeometryRenderer.ts:2]: I assume it is used in edit mode

Comment (albaranau) [reply src/features/focused_geometry_layer/renderers/FocusedGeometryRenderer.ts:2]: The black marker in edit mode comes from a different file (iconAtlas.png)

Comment (alesiahil) [reply src/features/focused_geometry_layer/renderers/FocusedGeometryRenderer.ts:2]: Done

File: src/core/localization/translations/en/common.json [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
   "to": "to",
-  "logout": "Logout",
+  "logout": "Log out",
   "save": "Save",
   "cancel": "Cancel",
   "create": "Create",
   "disasters": "Disasters",
   "loading": "Loading...",
-  "loading_events": "Loading events",
+  "loading_events": "Loading disasters",
```
Comment (Komzpa) [inline src/core/localization/translations/en/common.json:11]: @palina-krukovich review this change please.

Comment (palina-krukovich) [reply src/core/localization/translations/en/common.json:11]: looks good

**Review by palina-krukovich — APPROVED** [review]

File: src/core/localization/translations/en/common.json [hunk 3]
```diff
@@ -287,10 +287,10 @@
     "log_in": "Log in",
     "description": "Please log in to change your settings",
     "error": {
-      "email_empty": "Email has not to be empty!",
-      "email_invalid": "Email has to be valid!",
-      "password": "Password has not to be empty!",
-      "connect": "Couldn't connect to authentication service"
+      "email_empty": "Email cannot to be empty",
```
Comment (albaranau) [inline src/core/localization/translations/en/common.json]: "Email cannot be empty"

File: src/core/localization/translations/en/common.json [hunk 4]
```diff
@@ -287,10 +287,10 @@
...
+      "email_invalid": "Email has to be valid",
+      "password": "Password cannot to be empty",
```
Comment (albaranau) [inline src/core/localization/translations/en/common.json]: "Password cannot be empty"

File: src/core/localization/translations/en/common.json [hunk 5]
```diff
@@ -358,11 +358,11 @@
       "freehand_geometry": "Freehand geometry",
       "to_replace_reference_area": "To replace reference area, select an area on the map and click \"Save as reference area\" on toolbar.",
       "select_are_on_the_map": "Select an area on the map and click \"Save as reference area\" on toolbar to use it in further analysis.",
-      "set_the_reference_area": "Set the reference area"
+      "set_the_reference_area": "Set reference area"
     }
   },
   "current_event": {
-    "not_found_request": "Sorry, requested event not found."
+    "not_found_request": "Sorry, requested disaster is not found."
```
Comment (albaranau) [inline src/core/localization/translations/en/common.json]: Are we sure "is not found" works here? Present tense feels weird for some reason. Maybe "was not found" would work better, or just "not found"

Comment (Komzpa) [reply src/core/localization/translations/en/common.json]: Here are a few improved options for the message:

"Sorry, the requested disaster could not be found."
"Apologies, the requested disaster information is unavailable."
"Sorry, we couldn't locate the requested disaster."
"We apologize, but the disaster you requested is not found."
"The requested disaster information is not available. Please try again."

**Review by albaranau — APPROVED** [review]

File: src/core/localization/translations/en/common.json [hunk 6]
```diff
@@ -173,16 +173,16 @@
...
   },
   "advanced_analytics_panel": {
     "header_title": "Advanced analytics",
     "error": "Error while fetching advanced analytics data"
   },
   "advanced_analytics_empty": {
-    "not_found": "Sorry, requested event not found.",
-    "error": "An error occured!",
-    "analytics_for_selected": "Analytics for a selected area",
+    "not_found": "Sorry, the requested disaster was not found.",
```
Comment (alesiahil) [inline src/core/localization/translations/en/common.json]: @hevanskontur or @gmdowlingkontur can you take a look at this file https://github.com/konturio/disaster-ninja-fe/blob/9055f60ddff3a9bbc201f02584f250e16d57ede3/src/core/localization/translations/en/common.json (except 'about Disaster Ninja' part) and add your comments if there are any mistakes?

**Pull request merged**